packages/component-library/src/components/plmg-tooltip/plmg-tooltip.types.ts

```diff
@@ -0,0 +1,17 @@
+export const backgroundColors = ['neutral', 'primary'] as const;
+export type backgroundColors = typeof backgroundColors[number];
+export function isBackgroundColor(x: any): x is backgroundColors {
+  return backgroundColors.includes(x);
+}
+
+export const arrowSides = ['none', 'left', 'right', 'top', 'bottom'] as const;
+export type arrowSides = typeof arrowSides[number];
+export function isArrowSide(x: any): x is arrowSides {
+  return arrowSides.includes(x);
+}
+
+export const arrowPositions = ['start', 'middle', 'end'] as const;
+export type arrowPositions = typeof arrowPositions[number];
```

Comment:
Use unique names for these two declarations.
```suggestion
export const arrowPositions = ['start', 'middle', 'end'] as const;
export type PlmgTooltipArrowPosition = typeof arrowPositions[number];
```